From a61885a198a2f27ceeb4b0e776a77594c008f49f Mon Sep 17 00:00:00 2001
Message-Id: <patch-3.3-a61885a198-20210607T110044Z-avarab@gmail.com>
In-Reply-To: <cover-0.3-0000000000-20210607T110044Z-avarab@gmail.com>
References: <cover-0.3-0000000000-20210512T084137Z-avarab@gmail.com>
	<cover-0.3-0000000000-20210607T110044Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 5 May 2021 09:57:21 +0200
Subject: [PATCH v2 3/3] SubmittingPatches: remove pine-specific hints from MUA
 hints
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Replace the whole section about configuring pine with
no-strip-whitespace-before-send with a note that users should use
another MUA instead.

There hasn't been a release of Pine since late 2005, in Alpine the
no-strip-whitespace-before-send switch is already the default.

This documentation was initially added in 9740d289ee7 (Update
SubmittingPatches to add MUA specific notes., 2005-08-26). At this
point it's safe to say that it's not useful to anyone anymore, let's
remove it.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/SubmittingPatches | 54 ++-------------------------------
 1 file changed, 2 insertions(+), 52 deletions(-)

diff --git a/Documentation/SubmittingPatches b/Documentation/SubmittingPatches
index e372d17673..9c859dcfef 100644
--- a/Documentation/SubmittingPatches
+++ b/Documentation/SubmittingPatches
@@ -501,58 +501,8 @@ commit message.
 
 === Pine
 
-(Johannes Schindelin)
-
-....
-I don't know how many people still use pine, but for those poor
-souls it may be good to mention that the quell-flowed-text is
-needed for recent versions.
-
-... the "no-strip-whitespace-before-send" option, too. AFAIK it
-was introduced in 4.60.
-....
-
-(Linus Torvalds)
-
-....
-And 4.58 needs at least this.
-
-diff-tree 8326dd8350be64ac7fc805f6563a1d61ad10d32c (from e886a61f76edf5410573e92e38ce22974f9c40f1)
-Author: Linus Torvalds <torvalds@g5.osdl.org>
-Date:   Mon Aug 15 17:23:51 2005 -0700
-
-    Fix pine whitespace-corruption bug
-
-    There's no excuse for unconditionally removing whitespace from
-    the pico buffers on close.
-
-diff --git a/pico/pico.c b/pico/pico.c
---- a/pico/pico.c
-+++ b/pico/pico.c
-@@ -219,7 +219,9 @@ PICO *pm;
-	    switch(pico_all_done){	/* prepare for/handle final events */
-	      case COMP_EXIT :		/* already confirmed */
-		packheader();
-+#if 0
-		stripwhitespace();
-+#endif
-		c |= COMP_EXIT;
-		break;
-....
-
-(Daniel Barkalow)
-
-....
-> A patch to SubmittingPatches, MUA specific help section for
-> users of Pine 4.63 would be very much appreciated.
-
-Ah, it looks like a recent version changed the default behavior to do the
-right thing, and inverted the sense of the configuration option. (Either
-that or Gentoo did it.) So you need to set the
-"no-strip-whitespace-before-send" option, unless the option you have is
-"strip-whitespace-before-send", in which case you should avoid checking
-it.
-....
+Abandoned by its uptsream and has known flowed text bugs, use its
+successor alpine or another MUA instead.
 
 === Thunderbird, KMail, GMail
 
-- 
2.32.0.rc3.434.gd8aed1f08a7

